app/controllers/fhir/r4/api_controller.rb

```diff
@@ -490,12 +481,20 @@ def options
 
   private
 
+  # Handle general unkown error. Log and serve a 500
+  def handle_server_error(error)
+    Rails.logger.error(error.message)
+    error.backtrace.each { |line| Rails.logger.error line }
+    render json: operation_outcome_fatal.to_json, status: :internal_server_error
+  end
+
   # Check whether client is user or M2M flow, set instance variables appropriately.
   # Also return a 401 if user doesn't have API access
   def check_client_type
     return if doorkeeper_token.nil?
 
     if current_resource_owner.present?
+      Rails.logger.info("Client:User, ID:#{current_resource_owner.id}, Email:#{current_resource_owner.email}")
```

Comment:
don't need the parens for this (same goes for all of these logger calls)

Reply:
I personally prefer to use parens on the single param method calls (I just think it's easier to read), but fine with it either way. Updated here https://github.com/SaraAlert/SaraAlert/pull/707/commits/46afc9977cffe736d394ed995eb0a84b357cd3cd